<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPS Preview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="main_frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="jobShot_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="upper_h_line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="file_horizontalLayout">
        <item>
         <widget class="QLabel" name="file_label">
          <property name="text">
           <string>File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="file_lineEdit">
          <property name="mandatoryField" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="reset_toolButton">
          <property name="toolTip">
           <string>Update the file name output field with the current scene name.</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="res_range_horizontalLayout">
        <item>
         <widget class="QGroupBox" name="resolution_groupBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Resolution</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="resolution_comboBox">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Shot default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>150%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>125%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>110%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Render settings</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>75%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>25%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10%</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="res_horizontalLayout">
             <item>
              <widget class="QSpinBox" name="x_spinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>1920</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="resSep_label">
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="y_spinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>1080</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="range_groupBox">
          <property name="title">
           <string>Frame Range</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="range_comboBox">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Shot default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Timeline</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Current frame only</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="range_horizontalLayout">
             <item>
              <widget class="QSpinBox" name="start_spinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
               <property name="value">
                <number>1001</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="rangeSep_label">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="end_spinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
               <property name="value">
                <number>1100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="options_groupBox">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="offscreen_checkBox">
           <property name="text">
            <string>Offscreen</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="guides_checkBox">
           <property name="text">
            <string>Guides</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="launchViewer_checkBox">
           <property name="text">
            <string>Launch Viewer</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="noSelection_checkBox">
           <property name="text">
            <string>No Selection</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="slate_checkBox">
           <property name="text">
            <string>Slate</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="createQuicktime_checkBox">
           <property name="text">
            <string>Create QuickTime</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="lower_h_line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="preview_pushButton">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>jobShot_label</zorder>
     <zorder>upper_h_line</zorder>
     <zorder>options_groupBox</zorder>
     <zorder>lower_h_line</zorder>
     <zorder>preview_pushButton</zorder>
     <zorder>verticalSpacer_2</zorder>
     <zorder>verticalSpacer_3</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ui/rsc.qrc"/>
 </resources>
 <connections/>
</ui>
